contact.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: article
+titles:
+  # @start locale config
+  en      : &EN       Contact us
+  # @end locale config
+key: page-single
+---
+
+If you have questions or comments, please write to the
+<a href="mailto:">SDA Helpdesk</a>.
```

Comment:
Even if we don't have a correct helpdesk email, we should add _some_ address here I think, so it's not forgotten.

Reply:
Yes I agree. I'll check which one we can use.

Reply:
Fixed